### ISSUE TITLE
CSS - Methodology section on home page broken on large mobiles

### ISSUE THREAD

**Issue by bazzadp** [HTTPArchive/almanac.httparchive.org#389]
On my iPhone XS (can be replicated in Chrome dev tools by selecting "iPhone X" ) I see this on home page:

![image](https://user-images.githubusercontent.com/10931297/68525564-f842d400-02ca-11ea-8731-e15ecb4660cf.png)

Possible fix: knock down the padding on in [index.css](https://almanac.httparchive.org/static/css/index.css) from 40px to 30px, so from this

```
.methodology-data {
...
  padding: 20px 40px;
...
}
```
to this:

```
.methodology-data {
...
  padding: 20px 30px;
...
}
```

**Comment by rviscomi** [comment]
The padding is fine, but the `line-height` for these figures should be at least as large as the `font-size`.

**Comment by mikegeyser** [comment]
This will be a similar fix to #399, can I take this on?
